src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -82,7 +82,7 @@ public Navigator() : base()
         /// <summary>
         /// An event for the page disappearing signal which can be used to subscribe or unsubscribe the event handler provided by the user.
```

Comment:
description says about page disappearing but event name is "TransitionFinished" - is it the same?
also the description tells more about what event is than what this event is about - consider description like "An event fired when ..."

Reply:
Thank you I fixed.